### PR TITLE
[zh-cn] sync releases/patch-releases

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/releases/patch-releases.md

### CODE REVIEW

File: content/zh-cn/releases/patch-releases.md [hunk 1]
```diff
@@ -149,14 +149,14 @@ releases may also occur in between these.
...
-| December 2023         | 2023-12-08           | 2023-12-13  |
+| December 2023         | 2023-12-15           | 2023-12-19  |
 | January 2024          | 2024-01-12           | 2024-01-17  |
 | February 2024         | 2024-02-09           | 2024-02-14  |
 | March 2024            | 2024-03-08           | 2024-03-13  |
 -->
 | 月度补丁发布   | Cherry Pick 截止日期 | 目标日期      |
 |--------------|---------------------|-------------|
-| 2023 年 11 月 | 2023-12-08          | 2023-12-13  |
+| 2023 年 11 月 | 2023-12-15          | 2023-12-19  |
```
Comment (1000Delta) [inline content/zh-cn/releases/patch-releases.md:159]:
```suggestion
| 2023 年 12 月 | 2023-12-15          | 2023-12-19  |
```

**Review by 1000Delta — CHANGES_REQUESTED** [review]
LGTM

You can help to fix the month of release.

**Comment by utkarsh-singh1** [comment]
Seems to be a Netlify issue that timeout before build is complete, A simple empty commit may fix this issue.

**Comment by 1000Delta** [comment]
/lgtm

Thanks!

**Pull request merged**